### PR TITLE
Adding watex- machine learning research in water exploration package

### PR DESCRIPTION
**Pull request by WEgeophysics**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by WEgeophysics** [comment]
@conda-forge/help-python

**Comment by WEgeophysics** [comment]
@conda-forge-admin, please ping team¶

**Comment by WEgeophysics** [comment]
@conda-forge/staged-recipes

**Comment by WEgeophysics** [comment]
@conda-forge/staged-recipes All tests are passed. ``watex``  library is ready for review. Thanks

**Comment by WEgeophysics** [comment]
@conda-forge/help-python  
@conda-forge/staged-recipes 
All tests are passed. Ready for review. Thanks

File: recipes/watex/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/WEgeophysics/{{name}}/archive/refs/tags/v{{ version }}.tar.gz"
+  sha256: f5b2ab18cfc264b76faf1d16b9e06aa46025e63385c4e1f1208a9382cf19b616
+
+build:
+  number: 1
```
Comment (beckermr) [inline recipes/watex/meta.yaml:13]:
```suggestion
  number: 0
```

**Review by beckermr — CHANGES_REQUESTED** [review]
One item then it is ready to go!

**Review by beckermr — APPROVED** [review]

**Comment by WEgeophysics** [comment]
fix build number 

> One item then it is ready to go!

fix the build number and set it to 0. Thank you for your contribution.

**Pull request merged**